sulis_module/src/loot_list.rs

```diff
@@ -262,8 +245,9 @@ impl LootList {
                     }
                     Some(item) => item,
                 };
+                let identified = !adjectives.contains(&"unidentified".to_string());
```

Comment:
You should just be able to use `"unidentified"` directly here rather than `&"unidentified".to_string()`.  This is allocating memory to create a new string and then returning a reference to it, which is unnecessary.

However, `"unidentified"` should really be configurable as you have declared an unidentified item_adjective above.    I would put `unidentified_item_adjective` as a rule in rules.rs.

Reply:
I believe I needed this because "unidentified" is an &str, and this requires an &String. I'll try the rule.